v2/relay/resources.go

```diff
@@ -48,8 +51,9 @@ func DefaultResources() Resources {
 		MaxCircuits:     16,
 		BufferSize:      2048,
 
-		MaxReservationsPerIP:  4,
-		MaxReservationsPerASN: 32,
+		MaxReservationsPerPeer: 8,
+		MaxReservationsPerIP:   4,
```

Comment:
Do I understand correctly that with these limits there can be up to 8 registrations per 30 minutes by a single peer ID and 4 registrations per 30 minutes by a single IP address? If so, why set the former higher than the latter? I would expect multiple peers to use a single IP address (e.g. through NAT), but I doubt many peers use multiple IP addresses.

Reply:
yeah, we should probably reverse these numbers.

Reply:
Done.

Reply:
:+1:  Thanks!